app/models/archived_patient.rb

```diff
@@ -71,19 +71,14 @@ class ArchivedPatient
 
   mongoid_userstamp user_model: 'User'
 
-  # Archive & Delete patients who have fallen out of contact
-  def self.archive_dropped_off_patients!
-    Patient.where(:initial_call_date.lte => 1.year.ago).each do |patient|
-      ArchivedPatient.convert_patient(patient)
-      patient.destroy!
-    end
-  end
-
-  # Archive & Delete patients who are completely through our process
-  def self.archive_fulfilled_patients!
-    Patient.fulfilled_on_or_before(120.days.ago).each do |patient|
-      ArchivedPatient.convert_patient(patient)
-      patient.destroy!
+  # Archive & delete audited patients who called a several months ago, or any
+  # from a year plus ago
+  def self.archive_todays_patients!
```

Comment:
kind of nitpicky so please forgive me but what are your thoughts on naming this something like `archive_eligible_patients!` or something similar? `today` as an indicator feels like it could mean a few things to me, and I think that's a little more precise.